packages/venia-ui/lib/components/AccountInformationPage/accountInformationPage.gql.js

```diff
@@ -0,0 +1,62 @@
+import { gql } from '@apollo/client';
+
+export const SET_CUSTOMER_INFORMATION = gql`
+    mutation SetCustomerInformation(
+        $firstname: String!
+        $lastname: String!
+        $email: String!
+        $password: String!
```

Comment:
Instead of being verbose here, you can instead just validate to the mutation input type.
```suggestion
        $customerInput: CustomerInput!
```